index.bs

```diff
@@ -66,6 +66,88 @@ The primary goal is to make as much of this process as possible consistent betwe
 
 <h2 id="conventions">Conventions</h2>
 
+<h3 id="well-known-intrinsic-objects">Well-Known Intrinsic Objects</h3>
+
+Well-known intrinsics are built-in objects that are explicitly referenced by the algorithms of this specification and which usually have Realm specific identities. Unless otherwise specified each intrinsic object actually corresponds to a set of similar objects, one per Realm.
+
+Within this specification a reference such as %name% means the intrinsic object, associated with the current Realm, corresponding to the name. Determination of the current Realm and its intrinsics is described in ES2015, 8.3. The well-known intrinsics are listed in Table X.
+
+<table>
```

Comment:
The conventions section is only establishing spec conventions used in the rest of the spec, but it doesn't contain actual spec contents. So I wouldn't put this table here.

Since this table really cuts across different categories (especially across things that are universal vs browser-specific), I would put it in an appendix at the end, with a cell that refers to the section of the spec that introduces that row's intrinsic.